prime-router/build.gradle.kts

```diff
@@ -927,6 +927,10 @@ dependencies {
 
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.3")
 
+    // validations
+    implementation("com.networknt:json-schema-validator:1.4.0")
```

Comment:
no problem using this library - just wondering why we're not using jackson given the rest of the project already pulls that in? this thing uses jackson under the hood itself. why not cut out the middle man ?

Reply:
I brought this library in because I believe that Jackson can not validate using a json schema. Jackson can serialize/deserialize json (or yaml for us) and this library uses it for that purpose.

Reply:
I did some more digging and yes you're right. It generates but does not natively validate. I found this 

https://github.com/java-json-tools/json-schema-validator

but it seems as though the one you found is preferred these days 🍻